manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -763,7 +763,7 @@ properties:
   support_address: "http://support.cloudfoundry.com"
   description: null
   ssl:
-    skip_cert_verify: true
+    skip_cert_verify: false
```

Comment:
Good catch. Looks like this fixes a story we'd forgotten about: https://www.pivotaltracker.com/story/show/111678348

Could you pull it out into a separate PR for visibility and tag it with that story ID? Will it cause merge conflicts with this PR?